.travis.yml

```diff
@@ -13,7 +13,8 @@
           "postgresql"
         ],
         "install": [
-          "pip install -r requirements.txt"
+          "pip install -r requirements.txt",
+          "pip install coveralls"
```

Comment:
@blakes24 This is not part of the login feature. Make this a separate fix and create a separate pull request.

Reply:
ok